lib/src/credential_component.dart

```diff
@@ -1,24 +0,0 @@
-import 'package:angular/angular.dart';
```

Comment:
The credential component is removed. Its functionality (sign in and out) is moved to root component